kasenvr/project-athena
# Procedural Shader applies to entities other than target

Fixes https://github.com/kasenvr/project-athena/issues/166

Note: depends on the changes in https://github.com/kasenvr/project-athena/pull/276

Test plan:
- Start interface and go to the serverless domain with the default content.  Run [this](https://gist.github.com/HifiExperiments/ae9f742510012d465755d755affd94d4/raw/ccac684fa06b1c103b4ebdb838baf8228148637d/ShaderSwitching.js).
- You'll see a mannequin with a shader on its eyes.  On master, the shader will also apply to some of the other models around you.  Now, it'll only apply to the eyes.
- Press `o` to change the model and `i` to change the shader.  The shader should only ever apply to the model in front of you.
